### ISSUE TITLE
Notification Style - Daily Forecast Issue

### ISSUE THREAD

**Issue by HonestAbe7** [WangDaYeeeeee/GeometricWeather#323]
When I have the Daily Forecast set, I noticed no icons are shown to see the forecast for each day. Is this a known issue? All I see is the temperature highs and lows.

**Comment by papjul** [comment]
Please share a screenshot.

**Comment by HonestAbe7** [comment]
![Screenshot_20220521-162617](https://user-images.githubusercontent.com/88217236/169668003-c0809832-bc27-4781-b77a-ecdb1dc9c2bd.png)

**Comment by HonestAbe7** [comment]
There is my screenshot. Figured out it is volume down and the power button on my device. 😁

**Comment by papjul** [comment]
I don't think it's a bug because this would make a very big notification otherwise, but I will let the developer confirm. Maybe you can use widgets instead, so that you can keep your eye on daily forecast while still taking into account the prediction refreshes from the day?

**Comment by HonestAbe7** [comment]
Yes, that is what I am doing now. Although the weekday widget is so small and hard to read. Even increasing the font size takes away the high and lows from being seen. Just a minor issue though.
